guillotina/auth/utils.py

```diff
@@ -31,11 +30,6 @@
 
 
 def set_authenticated_user(user):
-    if user is not None:
```

Comment:
Instead of removing this and hardcoding `guillotina.Authenticated` into the anon user object, can we just make the `roles` not editable on the anon user object?

Reply:
done